casepro/msgs/models.py

```diff
@@ -417,6 +429,15 @@ def get_history(self):
         """
         return self.actions.select_related('created_by', 'label').order_by('-pk')
 
+    def get_lock(self, user_id):
```

Comment:
Hi @rowanseymour, we only use the ID in the function, and in one case we get the ID from context, https://github.com/praekelt/casepro/blob/6beca26817e493f388b6310be319aa97e2808d8b/casepro/msgs/views.py#L241, so we don't have the object.